tldraw/tldraw
# feat: add ungroup command

Adds the ability to ungroup shapes.

### Change type


### Test plan

1. Select a group of shapes
2. Run the ungroup command
3. Verify shapes are no longer grouped


### Release notes

- Added ungroup command
